code/__defines/species_languages.dm

```diff
@@ -59,6 +59,7 @@
 #define LANGUAGE_TERMINATOR "Hephaestus Darkcomms"	// HKs.
 
 // Lesser-form Languages
+#define LANGUAGE_BRAZILIAN "Gibbering"				// alberyk
```

Comment:
Going to agree with moon here. Rename to `LANGUAGE_GIBBERING` or something similar.